src/app/shared/card-content/card-content.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="card-text column pd-15">
+    <span class="sub-text full-white title ls-25">{{ title }}</span>
+    <span class="ai-start span yellow" [ngClass]="color">{{ value }}</span>
```

Comment:
May be better this way?
class="ai-start span yellow" [ngClass]="color" -> [ngClass]="['ai-start span yellow', color]"